Projects/Modules/Utility/Sources/Extensions/Extension+UIButton.swift

```diff
@@ -49,4 +41,13 @@ public extension UIButton {
         titleEdgeInsets = UIEdgeInsets(top: spacing, left: -image.size.width, bottom: -image.size.height, right: 0)
         imageEdgeInsets = UIEdgeInsets(top: -(titleSize.height + spacing), left: 0, bottom: 0, right: -titleSize.width)
     }
+
+    func addAction(for event: UIControl.Event = .touchUpInside, _ action: @escaping () -> Void) {
```

Comment:
```suggestion
    func addAction(for event: UIControl.Event = .primaryActionTriggered, _ action: @escaping () -> Void) {
```
touchUpInside도 문제 없긴한데 primaryActionTriggered라는거도 있어요

Reply:
오 처음 알았네요 감사합니다.